pynicotine/gtkgui/userbrowse.py

```diff
@@ -400,22 +408,19 @@ def on_file_row_activated(self, treeview, path, column):
 
     def on_file_popup_menu(self, menu, widget):
 
-        self.select_files()
```

Comment:
This line is still needed, otherwise there are no selected files for the context menu items.

Reply:
Thank you, this has greatly improved my understanding it's purpose, useful for implementation of new key bindings with multi-select actions that are now possible thanks to these experimentations and your help, which is much appreciated.